ansible/roles/stack-sunbird/templates/sunbird_groups-service.env

```diff
@@ -27,5 +27,6 @@ accesstoken.publickey.keyprefix={{ adminutil__access_keyprefix | default('access
 accesstoken.publickey.keycount={{ adminutil__access_keycount | default(10)}}
 sunbird_user_service_search_url=/private/user/v1/search
 sunbird_cs_search_url=/v3/search
-
+ENV_NAME={{env_name | default('dev')}}
+SUNBIRD_KAFKA_URL={{sunbird_kafka_url | default('localhost:9092')}}
```

Comment:
We dont have any such varaible named **sunbird_kafka_url**
We dont want to introduce new variables. Please reuse the variable **kafka_urls**
You can refer **ansible/roles/stack-sunbird/templates/sunbird_learner-service.env** for this

Reply:
updated